sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -74,12 +75,12 @@ export namespace ConnectionContext {
     return finalUserAgent;
   }
 
-  export function create(config: EventHubConnectionConfig, options?: ConnectionContextOptions): ConnectionContext {
+  export function create(config: EventHubConnectionConfig,  tokenProvider: TokenProvider, options?: ConnectionContextOptions): ConnectionContext {
```

Comment:
What do you think of adding `tokenProvider` to `ConnectionContextOptions` interface?
That way, most of the function signatures that changed in his PR need not change.